lib/dfe/reference_data/degrees/grades.rb

```diff
@@ -26,66 +28,77 @@ module Degrees
           'e2fe18d4-8655-47cf-ab1a-8c3e0b0f078f' =>
           { hesa_code: '02',
             name: 'Upper second-class honours (2:1)',
+            abbreviation: '2:1',
             suggestion_synonyms: [],
             match_synonyms: ['Upper second class honours', '2:1', '2.1'],
             group: :main_undergrad },
           '377a46ea-d6c6-4e87-9728-c1f0dd0ef109' =>
           { hesa_code: '03',
             name: 'Lower second-class honours (2:2)',
+            abbreviation: '2:2',
             suggestion_synonyms: [],
             match_synonyms: ['Lower second class honours', '2:2', '2.2'],
             group: :main_undergrad },
           '7f08cd7c-96d2-4706-9148-32468a6d67f9' =>
           { hesa_code: '04',
             name: 'Undivided second-class honours',
+            abbreviation: '2',
             suggestion_synonyms: [],
             match_synonyms: ['Undivided second class honours'],
             group: :other },
           '3d57e5f7-7904-4895-a982-ed953fff3f94' =>
           { hesa_code: '05',
             name: 'Third-class honours',
+            abbreviation: '3',
             suggestion_synonyms: [],
             match_synonyms: ['Third class honours', 'Third'],
             group: :main_undergrad },
           'da6f57ae-7de0-4215-82df-52322129e91f' =>
           { hesa_code: '06',
             name: 'Fourth-class honours',
+            abbreviation: '4',
             suggestion_synonyms: [],
             match_synonyms: ['Fourth class honours'],
             group: :other },
           '684d4a75-16ea-4c96-9a40-37cee18f1dec' =>
           { hesa_code: '07',
             name: 'Unclassified',
+            abbreviation: 'Unclassified',
             suggestion_synonyms: [],
             match_synonyms: [],
             group: :other },
           '171a98fc-280d-429a-9163-a380843adfce' =>
           { hesa_code: '08',
             name: 'Aegrotat',
+            abbreviation: 'Aegrotat',
             suggestion_synonyms: [],
             match_synonyms: [],
             group: :other },
           '4d3cc9d3-81dd-4457-ae2b-ee0bd298c51e' =>
           { hesa_code: '10',
             name: 'Ordinary degree',
+            abbreviation: 'Ordinary',
```

Comment:
```suggestion
            abbreviation: 'Ordinary',
```

I'm not sure this makes sense as an abbreviation.

Reply:
Does it make more sense now as a short_name?